test/diagnostics/Localization/Inputs/en.yaml

```diff
@@ -1,4 +1,4 @@
-#===--- fr.yaml - Localized diagnostic messages for French ---*- YAML -*-===#
+#===--- en.yaml - Localized diagnostic messages for English ---*- YAML -*-===#
```

Comment:
I think it's useful to have `fr` at least to make sure that something non-default works...